### PR TITLE
Add new sample scenes

### PR DESCRIPTION
**Pull request by shaynie**
## Changes
- Fixes: #11672 .

### CODE REVIEW

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/BoundsControlRuntimeExample.cs [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// This example script demonstrates various bounds control runtime configurations.
+    /// </summary>
+    public class BoundsControlRuntimeExample : MonoBehaviour
+    {
+        public TextMeshPro statusText;
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/BoundsControlRuntimeExample.cs:17]: summary for the public ones?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs [hunk 2]
```diff
@@ -0,0 +1,149 @@
+using System.Collections.Generic;
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs:1]: Add MIT license comment

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                InputDevices.GetDevicesWithCharacteristics(InputDeviceCharacteristics.HeldInHand, motionControllers);
+
+                if(motionControllers.Count > 0)
+                {
+                    SetVRModeActive();
+                }
+            }
+        }
+
+        public void SetVRModeActive()
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs:100]: add summary?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        {
+            interactorBehaviorControls.SetVR();
+
+            SetToggleActive(VRToggle, true);
+            SetToggleActive(hololensToggle, false);
+        }
+
+        public void SetHololensModeActive()
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/DisableInteractors.cs:108]: add summary?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ChangeRenderMode.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// This class allows to change the render mode on-the-fly. This is, for example, useful when trying to show a semi-transparent preview of an 
+    /// target.
+    /// </summary>
+    public static class ChangeRenderMode
+    {
+        public enum BlendMode
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ChangeRenderMode.cs:14]: Add summaries

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ChangeRenderMode.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            Opaque,
+            Cutout,
+            Fade,
+            Transparent
+        }
+
+        public static void ChangeRenderModes(Material standardShaderMaterial, BlendMode blendMode)
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ChangeRenderMode.cs:22]: Add summary

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/KeywordRecognitionHandler.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// Sample for binding speech recognition keywords to Unity Event functions.
+    /// </summary>
+    public class KeywordRecognitionHandler : MonoBehaviour
+    {
+        [Serializable]
+        public struct KeywordEvent
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/KeywordRecognitionHandler.cs:18]: Add summary

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/KeywordRecognitionHandler.cs [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+            [SerializeField]
+            public UnityEvent Event;
+        }
+
+        [Tooltip("List of speech recognition keywords and events to trigger.")]
+        [SerializeField]
+        private List<KeywordEvent> keywords = new List<KeywordEvent>();
+
+        public List<KeywordEvent> Keywords
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/KeywordRecognitionHandler.cs:31]: Add summary

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ScrollPanZoom/PanZoomBase.cs [hunk 9]
```diff
@@ -0,0 +1,591 @@
...
+
+        private bool zoomUsingHandsActive = false;
+        private Vector3 initialPalmPosition;
+        private XRNode handUsedToZoom;
+
+        [Tooltip("Toggles the direction of hand based zoom interactions")]
+        [SerializeField]
+        private bool invertPalmZoomDirection = true;
+
+        public override void ProcessInteractable(XRInteractionUpdateOrder.UpdatePhase updatePhase)
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/ScrollPanZoom/PanZoomBase.cs:298]: Add summary

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/UserInputRecorderFeedback.cs [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+        private void Update()
+        {
+            if (isShowingSomething && (DateTime.Now - startShowTime).TotalSeconds > maxShowDurationInSeconds)
+            {
+                ResetStatusText();
+            }
+        }
+
+        #region Data recording
+        public void StartRecording()
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/UserInputRecorderFeedback.cs:57]: Add summaries

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs [hunk 11]
```diff
@@ -0,0 +1,160 @@
+using Microsoft.MixedReality.Toolkit.Input;
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs:1]: Add MIT license comment

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+        [SerializeField]
+        private XRBaseInteractor[] grabInteractors;
+
+        [SerializeField]
+        private XRBaseInteractor[] pokeInteractors;
+
+        [SerializeField]
+        private XRBaseInteractor gazeInteractor;
+
+        public event Action<bool> onHandRayToggled;
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs:38]: summaries?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+            onGazeToggled?.Invoke(isActive);
+        }
+
+        public void SetHandPokeActive(bool isActive)
+        {
+            ToggleInteractors(pokeInteractors, isActive);
+            onPokeToggled?.Invoke(isActive);
+        }
+
+        public void SetHandGrabActive(bool isActive)
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs:90]: Summaries for these?

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/SolverExampleManager.cs [hunk 14]
```diff
@@ -0,0 +1,144 @@
...
+        {
+            SetRadialView();
+        }
+
+        public void SetTrackedHead()
+        {
+            TrackedType = TrackedObjectType.Head;
+        }
+
+        public void SetTrackedController()
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/SolverExampleManager.cs:55]: Summaries

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/SpeechKeywordRecognitionHandler.cs [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+
+using Microsoft.MixedReality.Toolkit.Subsystems;
+using UnityEngine;
+using UnityEngine.Events;
+using System.Collections.Generic;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    public class SpeechKeywordRecognitionHandler : MonoBehaviour
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/SpeechKeywordRecognitionHandler.cs:12]: Summary

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/TextToSpeechHandler.cs [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+
+        /// <summary>
+        /// Event raised when an error occurs. Contains the string representation of the error reason.
+        /// </summary>
+        [field: SerializeField]
+        public StringUnityEvent OnSpeakFaulted { get; private set; }
+
+        private TextToSpeechSubsystem textToSpeechSubsystem;
+
+        public void Speak()
```
Comment (marlenaklein-msft) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/TextToSpeechHandler.cs:56]: Add summary

**Review by marlenaklein-msft — CHANGES_REQUESTED** [review]

**Review by AMollis — APPROVED** [review]

**Review by AMollis — CHANGES_REQUESTED** [review]
Looks good... I agree with Marlena's comments.

For the samples, it's probably ok to leave some summary blocks out...but add the following to the top and bottom of the sample files if you do:

``
// Disable "missing XML comment" warning for sample. While nice to have, this documentation is not required for samples.
#pragma warning disable CS1591
``

``
#pragma warning restore CS1591
``

This will prevent warnings during the build process.

**Review by AMollis — APPROVED** [review]
I sign-off ... assuming @marlenaklein-msft  does as well

**Review by AMollis — APPROVED** [review]

**Review by marlenaklein-msft — APPROVED** [review]

**Pull request merged**